### ISSUE TITLE
Duplicate Identity issue when doing reconnect after network connection failure

### ISSUE THREAD

**Issue by imarshmultani** [twilio/twilio-video-app-react#226]
Hey there, 

What's happening is that Suppose if i try to switch the network and it takes me more then 20 seconds or 15 seconds, then i'm disconnected from the call and shown an error of either Signaling or media connection failed or media activity ceased. 

Then i have implemented an socket method on my server listens to it's events and checks if reconnection is successfully made, if yes then i get fresh data from sockets and without reloading the page as due to previous network error the twilio video room was disconnected i attempt to auto reconnect it , i do trigger connect of room and it does connect but within 1 second i'm disconnected and shown error of Duplicate participant identity, but i'm joining as same use who got disconnected previously and there is no other user with same identity. 

So basically when app fails the attempt to reconnect as network was not present for more then 20 seconds or so ... how i'm supposed to handle reconnection ? 

Please any suggestions would help to make fix issue or implement it properly will be appreciated .. 

Thanks
